Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -499,22 +508,30 @@ final class StatusTableViewController: ChartsTableViewController {
             
             self.tableView.beginUpdates()
             if let hudView = self.hudView {
-                // CGM Status HUD
+                // CGM Status
                 if let glucose = self.deviceManager.loopManager.glucoseStore.latestGlucose {
                     let unit = self.statusCharts.glucose.glucoseUnit
                     hudView.cgmStatusHUD.setGlucoseQuantity(glucose.quantity.doubleValue(for: unit),
                                                             at: glucose.startDate,
                                                             unit: unit,
                                                             staleGlucoseAge: self.deviceManager.loopManager.settings.inputDataRecencyInterval,
                                                             sensor: self.deviceManager.sensorState)
-
-                    if self.deviceManager.cgmManager != nil {
-                        hudView.cgmStatusHUD.presentStatusHighlight((self.deviceManager.cgmManager as? CGMManagerUI)?.cgmStatusHighlight)
-                    }
                 }
                 
-                // Pump Status HUD
-                if self.deviceManager.pumpManager != nil {
+                if let bluetoothStatusHighlight = self.bluetoothState.statusHighlight {
```

Comment:
[nit] seems management of the data shown by the HUDView would be in a _ViewModel_, not here in `StatusTableViewController`...but that's probably for a later refactor/cleanup

Reply:
While I agree, I'll leave this for a future refactor.